offload/cmake/caches/Offload.cmake

```diff
@@ -0,0 +1,12 @@
+set(LLVM_ENABLE_PROJECTS "clang;clang-tools-extra;lld" CACHE STRING "")
```

Comment:
FWIW, `compiler-rt` is also required sometimes, such as HIP.

Reply:
Why does HIP require the `clang` compiler-rt? Does it also enable `-frtlib=compiler-rt`? I can add it, but one unfortunate thing about having `compiler-rt` enabled in the main project is that it forces a CMake reconfigure *every* single time you run `ninja`.

Reply:
I'm not sure why `compiler-rt` is needed but it definitely doesn't work if it is not enabled.

> but one unfortunate thing about having compiler-rt enabled in the main project is that it forces a CMake reconfigure every single time you run ninja.

I always have it enabled but didn't see that issue.

Reply:
Only when it's enabled as a runtime.